defaults/main.yml

```diff
@@ -116,3 +121,16 @@ archivematica_src_automationtools_transfers_pidfile: "/var/archivematica/automat
 # Acceptance tests
 archivematica_src_acceptance_tests_browser_list: [ "Chrome" , "Firefox" ]
 archivematica_src_acceptance_tests_chromedriver_version: "2.29"
+
+# Configure AM
+# archivematica_src_configure_am_api_key & archivematica_src_configure_ss_api_key can be defined as vars. If not defined they will be autogenerated by the playbook
+archivematica_src_configure_ss_user: "test"                  # SS superuser
+archivematica_src_configure_ss_password: "test"              # SS superuser password
+archivematica_src_configure_ss_email: "test@example.com"     # SS superuser email address
+archivematica_src_configure_ss_url: "http://localhost:8000"  # SS url to register the pipeline
+archivematica_src_configure_am_user: "test"                  # Dashboard admin
+archivematica_src_configure_am_password: "test"              # Dashboard admin password
+archivematica_src_configure_am_email: "test@example.com"     # Dashboard admin password
+archivematica_src_configure_am_org_name: "test"              # Dashboard admin Org name
+archivematica_src_configure_am_org_id: "test"                # Dashboard admin Org id
```

Comment:
Can you add also the "First name" and "last name" fields? This way, we have all the configuration options exposed here.